### PR TITLE
DCI1 bitmap

### PR DESCRIPTION
**Pull request by WJKPK**
Branch include a function process bitmap (DCI 1 Format).

### CODE REVIEW

File: dciCommon.c [hunk 1]
```diff
@@ -206,3 +182,21 @@ uint32_t* dci_readValueFromDCI (uint64_t dci, const uint8_t bitLenghtOfDciParame
 	return outputArray;
 }
 
+uint8_t* dci1_bitmapDecoder(uint32_t bitmap, uint8_t bitmapBitLenght)
+{
+	uint8_t counter = bitmapBitLenght-1;
+	uint8_t* outputRBGIndex = malloc(sizeof(*outputRBGIndex));
+	for (uint8_t i = 0, j = 0; i < bitmapBitLenght; i++)
```
Comment (Zezwul) [inline dciCommon.c]: This for loop could be replaced by while

Comment (WJKPK) [reply dciCommon.c]: Gemacht

File: dciCommon.c [hunk 2]
```diff
@@ -153,34 +153,15 @@ void dci_print(char* output /*?*/)
...
-}
-
 static uint64_t createMask (const uint8_t n)
 {
-	uint64_t mask = 0;
+	uint64_t mask = 1;
 	if (n == 0)
 	{
-		return mask;
+		return mask = 0;
```
[the remark this answers is not in the document]
Comment (WJKPK) [reply dciCommon.c]: Gemacht

File: dciCommon.c [hunk 3]
```diff
@@ -118,3 +182,24 @@ uint32_t* dci_readValueFromDCI (uint64_t dci, const uint8_t bitLenghtOfDciParame
 	return outputArray;
 }
 
+uint8_t* dci1_bitmapDecoder(uint32_t bitmap, uint8_t bitmapBitLenght)
+{
+	uint8_t counter = bitmapBitLenght-1;
+	uint8_t* outputRBGIndex = malloc(sizeof(*outputRBGIndex));
+	uint8_t i, j = 0;
+	while(i < bitmapBitLenght)
```
Comment (Zezwul) [inline dciCommon.c]: Apply clean code (space after while)

Comment (WJKPK) [reply dciCommon.c]: Gemacht

**Pull request merged**